scala/lambdas/ingest-flow-control/src/main/scala/uk/gov/nationalarchives/ingestflowcontrol/Lambda.scala

```diff
@@ -122,16 +161,30 @@ class Lambda extends LambdaRunner[Input, Unit, Config, Dependencies] {
     }
 
     for {
+      _ <- IO(logger.info(Map("executionName" -> s"""${potentialInput.map(_.executionName)}""")(s"Starting flow control lambda with input: $potentialInput")))
```

Comment:
You've got your brackets a bit mixed up here as well. You're doing
```scala
logger.info(Map()("Something"))
```
Which means you're trying to get the value from the Map with a key of "Something" which isn't there so it throws an exception.

You need to change it to 
```scala
logger.info(Map())("Something")
```

All the logger calls are affected I think